docs/docs.json

```diff
@@ -23,17 +23,15 @@
         "pages": [
           "/how-to/run-agents",
           "/how-to/compose-agents",
-          "/how-to/build-agents",
-          "/how-to/import-agents",
-          "/how-to/share-agents"
+          "/how-to/import-agents"
         ]
       },
       {
         "group": "Core Concepts",
         "pages": ["/concepts/architecture"]
       },
       {
-        "group": "Agents",
+        "group": "Catalog",
```

Comment:
To me, catalog is a bit vague on the first look (not that Agents wouldn't be). Maybe Agents Catalog?

Also - if you decide to keep the word catalog, then the title do not much with the url see the line below.